diff --git a/dist/index.js b/dist/index.js
index 9baea0c8023c03d19b1494f818e1e11578f05b32..8f1a7a6050980d88e050423e63ef041195011fcc 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -101,7 +101,7 @@ function createRollupLicensePlugin(pluginOptions = {}, pluginName = "rollup-lice
       Object.entries(chunk.modules).filter(
         ([path, module]) => Boolean(path.match(/node_modules/)) && module.renderedLength > 0
       ).map(
-        ([path]) => path.startsWith("\0") ? path.replace(/^\0/, "") : path
+        ([path]) => (path.startsWith("\0") ? path.replace(/^\0/, "") : path).replace(/\\/g, '/')
       ).map((path) => [path, path.split("node_modules").pop()]).map(([path, filePath]) => {
         const segments = filePath.replace(/^\//, "").split("/");
         const packageName = segments[0].startsWith("@") ? `${segments[0]}/${segments[1]}` : segments[0];
