diff --git a/src/ng_package/packager.mjs b/src/ng_package/packager.mjs
index d7faa1dbc1bc5c8a61a4b3e90d306bb330c0b0d6..05b61eb36fa4124f6cf198b29bb8872e1260d5b3 100755
--- a/src/ng_package/packager.mjs
+++ b/src/ng_package/packager.mjs
@@ -8,7 +8,7 @@
 import * as fs from 'fs';
 import * as path from 'path';
 import { globSync } from 'tinyglobby';
-import { analyzeFileAndEnsureNoCrossImports } from './cross_entry_points_imports';
+import { analyzeFileAndEnsureNoCrossImports } from './cross_entry_points_imports.mjs';
 /**
  * List of known `package.json` fields which provide information about
  * supported package formats and their associated entry paths.
diff --git a/src/ng_package/rollup.config.js b/src/ng_package/rollup.config.js
index e819735986f746a3b24da9bd5c4f14220133ce60..9d731238981c42acabe19605b2bacd68eab1d246 100644
--- a/src/ng_package/rollup.config.js
+++ b/src/ng_package/rollup.config.js
@@ -159,7 +159,6 @@ const plugins = [
   nodeResolve({
     mainFields: ['es2020', 'es2015', 'module', 'browser'],
     jail: process.cwd(),
-    customResolveOptions: {moduleDirectory: nodeModulesRoot},
   }),
   stripBannerPlugin,
   commonjs({ignoreGlobal: true}),
