--- node_modules/rxjs/Observable.d.ts	2018-02-20 11:24:56.000000000 -0500
+++ node_modules/rxjs/Observable.d.ts	2018-02-20 11:25:21.000000000 -0500
@@ -69,6 +69,7 @@
     pipe<A, B, C, D, E, F, G>(op1: OperatorFunction<T, A>, op2: OperatorFunction<A, B>, op3: OperatorFunction<B, C>, op4: OperatorFunction<C, D>, op5: OperatorFunction<D, E>, op6: OperatorFunction<E, F>, op7: OperatorFunction<F, G>): Observable<G>;
     pipe<A, B, C, D, E, F, G, H>(op1: OperatorFunction<T, A>, op2: OperatorFunction<A, B>, op3: OperatorFunction<B, C>, op4: OperatorFunction<C, D>, op5: OperatorFunction<D, E>, op6: OperatorFunction<E, F>, op7: OperatorFunction<F, G>, op8: OperatorFunction<G, H>): Observable<H>;
     pipe<A, B, C, D, E, F, G, H, I>(op1: OperatorFunction<T, A>, op2: OperatorFunction<A, B>, op3: OperatorFunction<B, C>, op4: OperatorFunction<C, D>, op5: OperatorFunction<D, E>, op6: OperatorFunction<E, F>, op7: OperatorFunction<F, G>, op8: OperatorFunction<G, H>, op9: OperatorFunction<H, I>): Observable<I>;
+    pipe<R>(...operations: OperatorFunction<T, R>[]): Observable<R>;
     toPromise<T>(this: Observable<T>): Promise<T>;
     toPromise<T>(this: Observable<T>, PromiseCtor: typeof Promise): Promise<T>;
     toPromise<T>(this: Observable<T>, PromiseCtor: PromiseConstructorLike): Promise<T>;
